config/manager/manager.yaml

```diff
@@ -32,8 +32,8 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 30Mi
+            memory: 100Mi
```

Comment:
Please don't specify resource limits and requests before we have a proven performance issue (please remove them completely, even the default ones).

We did it before with `template-validator`, and it just made our life difficult when deploying on smaller clusters and debugging.

Reply:
Ok, I will remove them. I hit the OOM error with the default limits. On my machine the controller need about 56 Mi.

Reply:
Done